AK/StringBuilder.cpp

```diff
@@ -21,10 +21,11 @@ inline void StringBuilder::will_append(size_t size)
     Checked<size_t> needed_capacity = m_length;
     needed_capacity += size;
     VERIFY(!needed_capacity.has_overflow());
+    if (needed_capacity <= inline_capacity || needed_capacity.value() <= m_buffer.size())
```

Comment:
Maybe we should make `ByteBuffer::capacity` public and use that here?

Reply:
I wanted to change as few details as possible, but now that you suggest it: Yeah, will do that!